modules/storage-persistent-storage-pv.adoc

```diff
@@ -95,6 +95,19 @@ Direct matches are always attempted first. The volume's modes must match or cont
 
 All volumes with the same modes are grouped, and then sorted by size, smallest to largest. The binder gets the group with matching modes and iterates over each, in size order, until one size matches.
 
+ifndef::microshift[]
+[IMPORTANT]
+====
+Volume access modes describe volume capabilities. They are not enforced constraints. The storage provider is responsible for runtime errors resulting from invalid use of the resource. Errors in the provider show up at runtime as mount errors.
+
+ifndef::openshift-dedicated,openshift-rosa[]
+For example, NFS offers `ReadWriteOnce` access mode. If you want to use the volume's ROX capability, mark the claims as `ReadOnlyMany`.
+
+iSCSI and Fibre Channel volumes do not currently have any fencing mechanisms. You must ensure the volumes are only used by one node at a time. In certain situations, such as draining a node, the volumes can be used simultaneously by two nodes. Before draining the node, delete the pods that use the volumes.
+endif::openshift-dedicated,openshift-rosa[]
+====
+endif::microshift[]
+
 The following table lists the access modes:
```

Comment:
Since we're listing the different access mode and what they do we should also add ReadWriteMany & ReadOnlyMany.